README.md

```diff
@@ -12,6 +12,8 @@ Java idiomatic client for [Cloud Bigtable][cloud-bigtable].
 
 ## Quickstart
 
+WARNING: If you are experiencing version conflicts with gRPC, see [Version Conflicts](#version-conflicts).
```

Comment:
I dont think this is a warning, it's more of a tip. Maybe just add the sentence after the examples?

Reply:
Updated, not sure if that's what you wanted. If not, feel free to fix it and merge =)